blocks/subscriptions-block/components/ContactInfo/index.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useRef } from "react";
+
+import { usePhrases, Heading, Input, Button, Stack, Image } from "@wpmedia/arc-themes-components";
+
+import GoogleIcon from "./google.svg";
+import FacebookIcon from "./facebook.svg";
```

Comment:
There shouldn't be svgs included like this. For Facebook at least there's an icon in the `Icon` component. I can add a Google logo to the ticket to update that component (for the checkmark and the other missing icons) as well.

Reply:
@vgalatro I changed the Image for facebook, but for now I will keep the google until the icon on themes-components is done

Reply:
@vgalatro For that Google Icon ticket, just a heads up
https://developers.google.com/identity/branding-guidelines#incorrect-button-design
![image](https://github.com/WPMedia/arc-themes-blocks/assets/45465140/0b582ac7-2935-4e6e-a20f-f22d5baeb88b)

I noticed the v2 icons are all monochrome black & white. Google explicitly requires the full color version of the icon.

For Facebook, could we be provided a color version, with the Facebook blue? Facebook allows a black version of the logo, but only for technical reasons.
https://developers.facebook.com/docs/facebook-login/userexperience/#buttondesign
<img width="841" alt="Screenshot 2023-11-14 at 4 54 49 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/45465140/aa05d8a2-0a57-4cd3-846a-f97e17c8cebf">

LMK if I should file a ticket instead

Reply:
@mattkim93 , the v2 icons all accept a fill color as a parameter, so they can be any single color. If there is no fill set in the code they can also be styled with CSS. But you are right, we'll need a specific one for the multi-color Google logo. I've made an update in the ticket we're using to track that work.